README.adoc

```diff
@@ -31,24 +31,39 @@ us on Spectrum,link=https://spectrum.chat/opfab]
 == Introduction
 
 //tag::short_description[]
-OperatorFabric is a modular, extensible, industrial-strength and field-tested
-platform for use in electricity, water, and other utility operations.
+OperatorFabric is a modular, extensible, industrial-strength platform for use in electricity, water, and other utility operations.
 
-* System visualization and console integration
-* Precise alerting
-* Workflow scheduling
-* Historian
-* Scripting (ex: Python, JavaScript)
+It aims to facilitate operational activities for utilities in two ways :
+
+* Centralize realtime business events in a single place to avoid having multiple screens/software, operatorFabric provide 
+** events notification named "card"  with filtering features (severity , date , process ... )
+** events dispatching rules on a per user basis (using concepts of groups, organizational entities , process .. )
+** end points for business application to send the events. 
+** a mechanism of templating to customize events rendering (using HTML5 )
+** a view of the events on a timeline and agenda view 
+** storage of all the events (archive feature)
+** notifications via sounds 
+** possibilities  to integrate screen form other applications
+
+    
+* Facilitate interactions between operational control centers:
+** Share in real time information pre-formatted messages as card that can be manually send by operators or automatically by external solutions.
+** Introduce preformatted question/response exchange between control centers, this can be used to implement operational processes (with notion of last time to response).  
+** Share events in calendar (With capacity to have repeating events)
+
+In addition, the following features are available :  internalization, time-zone management, UI light/dark mode,  authorization mechanism.
+
+Integration with IT system is facilitate : support on Firefox and Chromium based browsers, docker deployment, communication with business application via API REST or Kafka, integration with external authentication systems (via OAuth2),  monitoring via Prometheus Endpoints.
```

Comment:
```suggestion
* Facilitate interactions between operational control centers:
** Share information in real time, as pre-formatted cards that can be sent either manually by operators or automatically by external solutions.
** Introduce pre-formatted question/response exchanges between control centers. This can be used to implement operational processes (with the notion of "last time to respond").  
** Share events in calendar (also allowing repeating events)

In addition, the following features are available :  internationalization, time-zone management, light/dark mode for the UI,  authorization mechanism.

Integration with existing IT systems is an overarching concern: support of Firefox and Chromium-based browsers, docker deployment, communication with business applications via REST API or Kafka, integration with external authentication systems (via OAuth2), monitoring via Prometheus endpoints.
```